### ISSUE TITLE
Asp.net docker page is misleading/confusing

### ISSUE THREAD

**Issue by StingyJack** [aspnet/Docs#4805]
https://github.com/aspnet/Docs/blob/master/aspnetcore/publishing/visual-studio-tools-for-docker.md

Title says it's for Visual Studio Tools for Docker, but it isn't. Where is that page?

Document doesn't maintain target platform and seems to interchange netfx and netcore. Which adds to the confusion when expecting info on Docker tools for Visual Studio. This is actually a wider problem than this specific article, having been left with half working tutorials for other msft tech.

I left comments on the document page on the msft doc site and they were deleted. I have a livefyre email telling me someone replied, but neither comment is present any longer. Why are they gone?

**Comment by Rick-Anderson** [comment]
@StingyJack  we've found livefyre to be unreliable.  Issues should be left here in the issue rather than with livefyre. Can you list the issues here? 

cc @scottaddie

**Comment by StingyJack** [comment]
> We've found livefyre to be unreliable
I see what you mean, its reffed me to 3 of my older posts in the last week and all have been removed. 

If you are asking about the specific issues, these are ones that are now gone.
![image](https://user-images.githubusercontent.com/15381181/32985749-76693fbe-cc8f-11e7-96b5-6a2da356dd49.png)

![image](https://user-images.githubusercontent.com/15381181/32985765-bf415e74-cc8f-11e7-9b1a-c6190c3f72ec.png)



The primary issue related to this post, is that I'm trying to get at least one of our programs on to a Docker image to see if its going to be a viable choice in the future, but nothing seems to be working correctly. So I went back to follow the directions on the page titled "Visual Studio Tools For Docker", and that only made things worse, as its actually "Using Asp.net Core with Docker". 

Fixing the page title and probably linking to the netfx version if one exists would probably help someone else find the right help. If that can be done, I would have no objection to closing this.

The other problem is that this conflation of netfx and netcore is not uncommon. The authors need to really make sure that they are not interchanging technologies, when the tech's are attempting parity but are still widely different.
